app/jobs/seekr.rb

```diff
@@ -80,13 +90,18 @@ def self.perform(slug, searchid)
 
     Dashing.send_event("#{slug}_twitter_mentions", {:comments => mentions}, :cache => true)
 
+    # Blacklist of Hashtags
+
+    blacklist = JSON.parse(File.read(Rails.root.join("app/jobs/hashtags.json")))['hashtags']
```

Comment:
O Padrão I18n do Rails também utiliza o mesmo padrão.

Como é possível ver em baixo, 


# The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
# config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
# config.i18n.default_locale = :de

Creio que não é um problema